sim/gui.py

```diff
@@ -249,22 +250,12 @@ def draw_maze(
             robot_direction (Direction): Direction robot is facing. Defaults to north.
             heatmap (bool): Draw heatmap on maze. Defaults to False.
         """
-        # Draw robot's route before all other maze elements:
-        for i, cell in enumerate(maze.route):
-            x1 = cell[1] * tile_size + offset[0]
-            y1 = cell[0] * tile_size + offset[1]
-            if i + 1 < len(maze.route):
-                next_cell = maze.route[i + 1]
-                x2 = next_cell[1] * tile_size + offset[0]
-                y2 = next_cell[0] * tile_size + offset[1]
-                self.draw_route_cell(Position(x1, y1), Position(x2, y2), 'magenta')
-
         wall_color = pg.Color(self.wall_color)
-        for row, col, walls in maze:
+        robot_position = Position(0, 0)
```

Comment:
`robot_position` is a confusing name.
Also, maybe start with `None` and `assert` later, `(0, 0)` is not a good screen position...